dataprofiler/profilers/profile_builder.py

```diff
@@ -2087,12 +2096,13 @@ def _update_row_statistics(
                 null_in_row_count = null_in_row_count.union(null_row_indices)
 
         # If sample_ids provided, increment since that means only new data read
-        if sample_ids is not None:
-            self.row_has_null_count += len(null_in_row_count)
-            self.row_is_null_count += len(null_rows)
-        else:
-            self.row_has_null_count = len(null_in_row_count)
-            self.row_is_null_count = len(null_rows)
+        if self.options.row_statistics.null_count.is_enabled:
```

Comment:
does this condition check belong up at line 2062? if null count is disabled, do we need to any code below 2062?